app/controllers/actions/PrivilegedAction.scala

```diff
@@ -16,64 +16,55 @@
 
 package controllers.actions
 
-import config.{AppConfig, UnauthorizedReviewErrorHandler}
-import javax.inject.Inject
 import models.requests.IdentifierRequest
 import play.api.mvc.Results._
 import play.api.mvc._
-import play.api.{Configuration, Environment, Logger}
-import uk.gov.hmrc.auth.core.AuthProvider.PrivilegedApplication
+import play.api.Logger
+import uk.gov.hmrc.auth.core.authorise.Predicate
 import uk.gov.hmrc.auth.core._
-import uk.gov.hmrc.auth.core.retrieve.{Credentials, Name, ~}
+import uk.gov.hmrc.auth.core.authorise.Predicate
```

Comment:
Duplicate ?

Reply:
yes, the first only imports things immediately in core, so things under retrieve and authorise need to be done separately

Reply:
Optimised - resolved